AdevintaSwiftUIPager.podspec

```diff
@@ -1,23 +1,23 @@
 Pod::Spec.new do |s|
 
-  s.name         = "SwiftUIPager"
-  s.version      = '1.13.0'
+  s.name         = "AdevintaSwiftUIPager"
+  s.version      = '2.0.0'
```

Comment:
The latest version should be 2.1.0

Reply:
I think that from now on we should own the release versions, and not depend in the official one. So I was thinking that our renamed fork would start with the 2.0.0, and then once we update with the official one, we can bump it to 3.0.0. What do you think?